workers/deployment_manager/DeploymentManager/Worker/DeploymentManager.cs

```diff
@@ -0,0 +1,193 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Improbable.SpatialOS.Deployment.V1Alpha1;
+using Improbable.Worker;
+using Improbable.Worker.Alpha;
+using Deployment = Improbable.SpatialOS.Deployment.V1Alpha1.Deployment;
+
+namespace DeploymentManager
+{
+    internal class DeploymentManager
+    {
+        public static string WorkerType = "DeploymentManager";
+
+        private string projectName;
+        private string assemblyName;
+        private string deploymentNamePrefix;
+        private string devAuthToken;
+        private int numberOfDeployments;
+        private string clientType;
+        private int maxPlayerCount;
+        private int playerCount;
+
+        private Connection metaConnection;
+
+        public DeploymentManager(string receptionistHost, ushort receptionistPort, string workerId, string projectName, string clientType,
+            string assemblyName, string deploymentNamePrefix, int numberOfDeployments, int maxPlayerCount)
+        {
+            this.projectName = projectName;
+            this.assemblyName = assemblyName;
+            this.deploymentNamePrefix = deploymentNamePrefix;
+            this.numberOfDeployments = numberOfDeployments;
+            this.clientType = clientType;
+            this.maxPlayerCount = maxPlayerCount;
+
+            var serviceAccountToken = GetFileContent("ServiceAccountToken.txt");
```

Comment:
It may be best to include this file as an embedded resource in the executable, rather than reading it from disk.